todo_app/tests/integration/integration_test.py

```diff
@@ -24,7 +24,7 @@ def json(self):
 def stub(url, params={}):
     test_board_id = os.environ.get('TRELLO_BOARD_ID')
```

Comment:
We can get rid of this variable if we're just assuming the value directly in the string!